[OSDOCS#17160] using-ruby CQA (#117479)
openshift/openshift-docs: modules/templates-rails-configuring-application.adoc

[id="templates-rails-configuring-application_{context}"]
= Configuring application for {product-title}

[role="_abstract"]

> You can edit the `default` section in the `config/database.yml` file to configure your Rails application for {product-title}. This lets {product-title} supply the correct database credentials at runtime so your application can connect to PostgreSQL on the cluster.

Reviewer: ```suggestion
To configure your Rails application for {product-title}, you must edit the `default` section in the `config/database.yml` file. This is required for {product-title} to supply the correct database credentials at runtime so your application can connect to PostgreSQL.
```
